concordium-node/src/consensus_ffi/helpers.rs

```diff
@@ -249,6 +261,18 @@ impl ConsensusFfiResponse {
             Success | PendingFinalization | Asynchronous => true,
         }
     }
+
+    /// Get the label. This is used when updating metrics of the prometheus
+    /// exporter.
+    pub fn label(&self) -> &str {
```

Comment:
Do we want more fine grained labels? I.e. 
`is_acceptable`, `is_pending`, `is_rebroadcastable` .

Reply:
If you think it would be useful information when maintaining a node, we should add it, but if not, then it would just add more for the maintainer to understand.

Reply:
Let's go with this for now. If we find the need we can revise in the future.